app/models/supplier.rb

```diff
@@ -11,6 +11,12 @@ class Supplier < ApplicationRecord
 
   has_many :services, dependent: :destroy
 
+  after_create :send_supplier_email
+
+  def send_supplier_email
+    SupplierMailer.new_supplier_waiting_for_approval(self).deliver
```

Comment:
You can use `.deliver_later` instead of .`deliver` for a better and fast response of your app